pkg/capabilities/consensus/ocr3/models.go

```diff
@@ -0,0 +1,27 @@
+package consensus
+
+import (
+	"context"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type request struct {
+	Observations      values.Value `mapstructure:"-"`
+	AggregationMethod string
```

Comment:
I'm assuming that you're leaving this for now? If so maybe just add a comment that this will be extended or moved.

Reply:
I've actually removed it :) I thought I'd forgotten to address some feedback.